### ISSUE TITLE
Notification Expiration doesn't work on iOS

### ISSUE THREAD

**Issue by ClayYahuira** [AvaloniaUI/Avalonia#14516]
## Describe the bug
Notification expiration TimeSpan is not working on iOS.

## To Reproduce

1. SetUp the WindowNotificationManager
2. Create a simple notification with the default Expiration.
3. Show the Notification.
4. The Notification doesn't disappear in 5 seconds(default).

## Expected behavior
The notification should hide like on desktop or android after the expiration time.

## Environment 

- OS:  iOS
- Avalonia-Version: 11.0.6

## Additional context
This is how I set up the NotificationService:
        
```
public void Initialize(object? visual)
        {
            var topLevel = TopLevel.GetTopLevel(visual as Visual);
            _windowNotificationManager = new WindowNotificationManager(topLevel)
            {
                Position = NotificationPosition.BottomRight,
                MaxItems = 1,
            };
        }
```

How I create my notification:

```
public object GetComingSoonNotification()
        {
            return new Notification(
                title: string.Empty,
                message: _languageService.GetResourceFromName(COMING_SOON_MESSAGE));
        }
```

How I show the notification:
 
```
public async Task ShowNotification(object? notification)
        {
            if (notification is INotification notificationObject)
            {
                    _windowNotificationManager?.Show(notificationObject);
            }
        }
```

This same code is working well on android but on iOS the Expiration is not applying and I need to tap over any kind of button in order to hide the notification.

**Comment by jp2masa** [comment]
Probably fixed by #13942, try upgrading to 11.0.7.

**Comment by maxkatz6** [comment]
Yes, it was fixed in 11.0.7

**Comment by ClayYahuira** [comment]
I tried with the 11.0.7 and the bug was solved. Nice job 👍
